src/browser/components/MattermostView.jsx

```diff
@@ -126,8 +126,14 @@ export default class MattermostView extends React.Component {
           shell.openExternal(e.url);
         }
       } else {
-        // if the link is external, use default os' application.
-        ipcRenderer.send('confirm-protocol', destURL.protocol, e.url);
+        const parsedURL = Utils.parseURL(e.url);
```

Comment:
not a comment to change this, but we should unify url and URL uses into a single lib (there is a ticket suggesting that)